### DISCUSSION TITLE
Branch theme-system

### DISCUSSION THREAD

**Discussion by lucasfth** [PhilipFlyvholm/learnit-plus-plus#15] [Q&A] [answered]
Hey @PhilipFlyvholm.
I just saw that you had a branch named theme-system.
Is that regarding the potential feature we talked about in #4.
Specifically, you wrote the following part:

> The color changes look good, however, it contradicts the current color changes made by the extension. A possible solution would be creating a theming system, where the user can choose the themes.

It is just so I know if I can start working on the colors which mimicked the old style or not.

**Comment by PhilipFlyvholm** [comment, marked as answer]
Yes, this is the theming system I mentioned. There is still work to be done to get a way to select the themes, but a theme can be registered in the `src/styles/main.ts`.
The `main.css` is used for features/improvements that are non theme related and then a folder can be created for each theme.

**Reply by lucasfth** [reply]
Nice, that is really cool. I will add the theming I have created earlier to the folder
